examples/plugins/example_launcher_plugin/tests/test_example_launcher_plugin.py

```diff
@@ -32,6 +38,9 @@ class TestExampleLauncher(LauncherTestSuite):
     "task_launcher_cfg, extra_flags",
     [({}, ["-m", "hydra/launcher=example"])],
 )
+@mark.filterwarnings(
+    "ignore:provider=example-searchpath-plugin, path=arbitrary_package.conf is not available:UserWarning"
+)
 class TestExampleLauncherIntegration(IntegrationTestSuite):
```

Comment:
Can you explain why all those example plugins has this warning?
example-searchpath-plugin is adding something that - as far as I know - is a valid Python module that should be available once it's installed and should not trigger any warnings.